README.md

```diff
@@ -67,7 +67,7 @@ gaiad version
 You should see:
 
 ```
-0.18.0-dev-af15f89
+0.18.0-dev-af15f895
```

Comment:
This depends on system git settings I believe. On Ubuntu, its 7 characters by default.